src/components/ResultsView.tsx

```diff
@@ -386,6 +386,7 @@ const ResultsView: ResultsViewComponent = ({
             </Tooltip>
           )}
           <Popover
+            placement="left-end"
```

Comment:
I also snuck this in here to deal with the placement of `Edit Query` being unclickable

![image](https://github.com/dvargas92495/roamjs-query-builder/assets/3792666/69764e0b-ff00-4a0a-ba9e-5b4c40aad4f2)

Reply:
how does this behave if the qb is at the bottom of the page? I guess you could always scroll down further

Reply:
In my tests, worst case scenario is `Copy Query` is hidden when there are no results.  But a user can also toggle footer to alleviate that.

https://github.com/dvargas92495/roamjs-query-builder/assets/3792666/8e405057-63a8-4064-b1c0-7e4c6b351f6e